src/common/serviceInstanceHelpers.js

```diff
@@ -13,7 +17,7 @@ class DefaultServiceInstanceTransform {
       },
       spec: {
         clusterServiceClassExternalName: siInfo.name,
-        clusterServicePlanExternalName: `default-${siInfo.name}`
+        clusterServicePlanExternalName: `default`
```

Comment:
@witmicko Can we revert the change to this? This is causing the provison of che, launcher and fuse to fail because the plan name we currently have to the broker is in this format: `default-${siInfo.name}`